app/src/main/java/com/github/se/eventradar/model/repository/event/FirebaseEventRepository.kt

```diff
@@ -6,16 +6,20 @@ import com.google.firebase.firestore.CollectionReference
 import com.google.firebase.firestore.FirebaseFirestore
 import com.google.firebase.firestore.ktx.firestore
 import com.google.firebase.ktx.Firebase
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
 import kotlinx.coroutines.channels.awaitClose
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.callbackFlow
 import kotlinx.coroutines.tasks.await
 
 class FirebaseEventRepository(db: FirebaseFirestore = Firebase.firestore) : IEventRepository {
   private val eventRef: CollectionReference = db.collection("events")
+  private val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm")
```

Comment:
Suggestion:
Instead of calling `val currentDateTimeString = LocalDateTime.now().format(formatter)` in multiple functions,
consider adding : `private val currentDateTimeString: String get() = LocalDateTime.now().format(formatter)`